<launch>
    <node name="usb_joystick" pkg="joy" type="joy_node" output="screen" required="false"
        clear_params="true">
        <param name="dev"
            value="/dev/input/js1"
            type="str" />
        <param name="deadzone" value="0.05" />
        <param name="autorepeat_rate" value="30.0" />
    </node>

    <node pkg="bw_teleop" type="simple_joystick_relay.py" name="simple_joystick_relay"
        output="screen"
        clear_params="true">
        <param name="axis_linear" value="0" />
        <param name="axis_angular" value="1" />
        <param name="scale_linear" value="5.0" />
        <param name="scale_angular" value="20.0" />
        <remap from="cmd_vel" to="cmd_vel/teleop" />
    </node>
</launch>